### ISSUE TITLE
Make the OAuth2 landing page pretty

### ISSUE THREAD

**Issue by mickmister** [mattermost/mattermost-plugin-mscalendar#94]
Currently the page is plain text, but it should be stylized like the Jira plugin's OAuth complete page. The page should also offer Disconnect, just as the Jira plugin's page does.

**Comment by mickmister** [comment]
@hanzei The reason I had some of these on the board was to be able to easily browse which tasks have or have not been started in each milestone. Do you think there is some value in having tickets on the board that have not been assigned yet?

**Comment by hanzei** [comment]
Not really. The idea of the board is to keep an overview on what people are doing right now. Maybe you you bookmark the milestone you care about?

**Comment by mickmister** [comment]
@hanzei It was helpful in a release discussion meeting when we were determining which tickets should be in which release. Browsing from the board view was more convenient than the list view that the milestone page gives you. We were able to filter on milestone and assignee, and see the different progresses on each issue, and adjust milestones/assignees all on one page.

I personally don't think we would be polluting the board as we typically filter by assignee during triage. The only time I would be using it would be for release planning, which doesn't happen that often, but it was definitely helpful.

**Comment by aaronrothschild** [comment]
@abhijit-singh can you sketch out what a Mattermost HTML landing page should say/look like after a user has authenticated with MSCalendar/Teams/Zoom/Jira/Github/etc.  It would be great if we could make a standard template one that would be easy to modify for each plugin/app.   

- it should tell them to run the help command
- or help them get started
- reassure them they connected
- can close this window now
-
